go/engine/track_test.go

```diff
@@ -202,6 +202,10 @@ func TestTrackNewUserWithPGP(t *testing.T) {
 
 // see issue #578
 func TestTrackRetrack(t *testing.T) {
+
+	// XXX - can this test be salvaged?
+	t.Skip()
+
```

Comment:
i'd say either we nuke it or change it to try tracking logged out or something.  

it's checking on line 246 for a GetPassphrase call, which will never work on mobile.